libs/input-mapping/azure-pipelines.tests.yml

```diff
@@ -0,0 +1,96 @@
+jobs:
+  - template: ../../azure-pipelines/jobs/run-tests.yml
+    parameters:
+      jobName: cs
+      displayName: Code Style
+      serviceName: ci-input-mapping
+      testCommand: composer phpcs
+
+  - template: ../../azure-pipelines/jobs/run-tests.yml
+    parameters:
+      jobName: aws_tests_php74_commonPart1
+      displayName: Test on AWS backend CommonPart1 (PHP 7.4)
+      serviceName: ci-input-mapping
+      testCommand: /code/libs/input-mapping/vendor/bin/phpunit --testsuite CommonPart1; \
+        /code/libs/input-mapping/vendor/bin/phpunit --testsuite Aws
```

Comment:
@joker806 btw, I had to do this because these tests cannot run in parallel.  Probably not the ideal way to do it though

Reply:
yeah, that doesn't work at all lol.  I will use the depends on for this too